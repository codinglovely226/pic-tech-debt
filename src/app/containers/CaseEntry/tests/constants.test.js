import {
  ADD_CALL_RECORD,
  ADD_CASE,
  END_CALL_RECORD,
  RESET_AGENT_VIEW_CENTRE_NAME,
  UPDATE_AGENT_VIEW_CENTRE_NAME,
  UPDATE_APPLICATION_KEY_ID,
  UPDATE_CASE_OUTCOME,
  UPDATE_CASE_STATUS,
  UPDATE_CASE_TOPIC,
  UPDATE_CASE_TYPE,
  UPDATE_TEMPLATE_TOPIC,
  UPDATE_TEMPLATE_CASE_TYPE,
  LINK_CASE_INCIDENT,
  UNLINK_CASE_INCIDENT,
  UPDATE_INCIDENT_NAME,
} from '../constants';

describe('App action constants', () => {
  it('ADD_CALL_RECORD', () => {
    expect(ADD_CALL_RECORD).toMatchSnapshot();
  });
  it('ADD_CASE', () => {
    expect(ADD_CASE).toMatchSnapshot();
  });
  it('END_CALL_RECORD', () => {
    expect(END_CALL_RECORD).toMatchSnapshot();
  });
  it('RESET_AGENT_VIEW_CENTRE_NAME', () => {
    expect(RESET_AGENT_VIEW_CENTRE_NAME).toMatchSnapshot();
  });
  it('UPDATE_AGENT_VIEW_CENTRE_NAME', () => {
    expect(UPDATE_AGENT_VIEW_CENTRE_NAME).toMatchSnapshot();
  });
  it('UPDATE_APPLICATION_KEY_ID', () => {
    expect(UPDATE_APPLICATION_KEY_ID).toMatchSnapshot();
  });
  it('UPDATE_CASE_OUTCOME', () => {
    expect(UPDATE_CASE_OUTCOME).toMatchSnapshot();
  });
  it('UPDATE_CASE_STATUS', () => {
    expect(UPDATE_CASE_STATUS).toMatchSnapshot();
  });
  it('UPDATE_CASE_TOPIC', () => {
    expect(UPDATE_CASE_TOPIC).toMatchSnapshot();
  });
  it('UPDATE_CASE_TYPE', () => {
    expect(UPDATE_CASE_TYPE).toMatchSnapshot();
  });
  it('UPDATE_TEMPLATE_TOPIC', () => {
    expect(UPDATE_TEMPLATE_TOPIC).toMatchSnapshot();
  });
  it('UPDATE_TEMPLATE_CASE_TYPE', () => {
    expect(UPDATE_TEMPLATE_CASE_TYPE).toMatchSnapshot();
  });
  it('LINK_CASE_INCIDENT', () => {
    expect(LINK_CASE_INCIDENT).toMatchSnapshot();
  });
  it('UNLINK_CASE_INCIDENT', () => {
    expect(UNLINK_CASE_INCIDENT).toMatchSnapshot();
  });
  it('UPDATE_INCIDENT_NAME', () => {
    expect(UPDATE_INCIDENT_NAME).toMatchSnapshot();
  });
});
