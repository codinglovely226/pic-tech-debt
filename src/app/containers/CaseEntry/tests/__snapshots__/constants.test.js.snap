exports[`App action constants ADD_CALL_RECORD 1`] = `"pic/CaseEntry/ADD_CALL_RECORD"`;

exports[`App action constants ADD_CASE 1`] = `"pic/CaseEntry/ADD_CASE"`;

exports[`App action constants END_CALL_RECORD 1`] = `"pic/CaseEntry/END_CALL_RECORD"`;

exports[`App action constants LINK_CASE_INCIDENT 1`] = `"pic/CaseEntry/LINK_CASE_INCIDENT"`;

exports[`App action constants RESET_AGENT_VIEW_CENTRE_NAME 1`] = `"pic/CaseEntry/RESET_AGENT_VIEW_CENTRE_NAME"`;

exports[`App action constants UNLINK_CASE_INCIDENT 1`] = `"pic/CaseEntry/UNLINK_CASE_INCIDENT"`;

exports[`App action constants UPDATE_AGENT_VIEW_CENTRE_NAME 1`] = `"pic/CaseEntry/UPDATE_AGENT_VIEW_CENTRE_NAME"`;

exports[`App action constants UPDATE_APPLICATION_KEY_ID 1`] = `"pic/CaseEntry/UPDATE_APPLICATION_KEY_ID"`;

exports[`App action constants UPDATE_CASE_OUTCOME 1`] = `"pic/CaseEntry/UPDATE_CASE_OUTCOME"`;

exports[`App action constants UPDATE_CASE_STATUS 1`] = `"pic/CaseEntry/UPDATE_CASE_STATUS"`;

exports[`App action constants UPDATE_CASE_TOPIC 1`] = `"pic/CaseEntry/UPDATE_CASE_TOPIC"`;

exports[`App action constants UPDATE_CASE_TYPE 1`] = `"pic/CaseEntry/UPDATE_CASE_TYPE"`;

exports[`App action constants UPDATE_INCIDENT_NAME 1`] = `"pic/CaseEntry/UPDATE_INCIDENT_NAME"`;

exports[`App action constants UPDATE_TEMPLATE_CASE_TYPE 1`] = `"pic/CaseEntry/UPDATE_TEMPLATE_CASE_TYPE"`;

exports[`App action constants UPDATE_TEMPLATE_TOPIC 1`] = `"pic/CaseEntry/UPDATE_TEMPLATE_TOPIC"`;
